developers/submit-data.md

```diff
@@ -174,7 +174,7 @@ willingness to accept changes to fix this issue.
 
 ## API
 
-Users can currently create and submit `BlobTx`s in four ways.
+Users can currently create and submit `BlobTx`s in six ways.
```

Comment:
did this PR add 2 ways, or is the old version wrong? thanks!

Reply:
> did this PR add 2 ways, or is the old version wrong? thanks!

There were five methods listed already, not four.